lightcycle-lib/src/main/java/com/soundcloud/lightcycle/util/LightCycleBinderHelper.java

```diff
@@ -0,0 +1,28 @@
+package com.soundcloud.lightcycle.util;
+
+import com.soundcloud.lightcycle.LightCycleDispatcher;
+import com.soundcloud.lightcycle.LightCycles;
+
+public class LightCycleBinderHelper {
+
+    private final LightCycleDispatcher<?> dispatcher;
+    private boolean isBound = false;
+
+    LightCycleBinderHelper(LightCycleDispatcher<?> dispatcher) {
+        this.dispatcher = dispatcher;
+    }
+
+    public void bindIfNecessary() {
+        if (!isBound) {
+            LightCycles.bind(dispatcher);
+            isBound = true;
+        }
+    }
+
+    public static class Factory {
```

Comment:
Instead of the factory class we could just have a `public static LightCycleBinderHelper create(...)` method in the `LightCycleBinderHelper` directly.